source/Handlebars/Arguments.cs

```diff
@@ -58,7 +58,7 @@ public Arguments(object arg1) : this()
             Length = 1;
         }
         
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        // [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Why do you need to comment out the attribute?

Reply:
Sorry, this was a debugging leftover.